packages/liveblocks-core/src/client.ts

```diff
@@ -666,8 +813,14 @@ export function createClient<U extends BaseUserMeta = DU>(
         getRoomIds() {
           return Array.from(roomsById.keys());
         },
+
+        // "All" threads (= "user" threads)
         getThreads,
         getThreadsSince,
+
+        // "Room" threads (= "normal" threads)
+        getRoomThreads,
+        getRoomThreadsSince,
```

Comment:
I don't like all these variants here. Could we DRY them up into one?

Reply:
Agreed. One seems to be coming from `notifications` related export (not sure if it is a fitting place). I will look into the efforts required to consolidate the two. Good point!